src/v1.ts

```diff
@@ -43,18 +67,48 @@ export function NewClientWithCustomCert(
 
 /**
  * NewClientWithChannelCredentials creates a new client for calling Authzed APIs using custom grpc ChannelCredentials.
+ * 
+ * The combined client exposes all service-level methods at the root which directly call grpc1-generated methods
+ * while also exposing a `promises` object containing all promise-wrapped methods.
```

Comment:
Include a note that promise versions of methods that returns response streams will buffer all results before resolving the promise.